libs/requestutils/requestutils.go

```diff
@@ -44,6 +45,9 @@ func Read(ctx context.Context, body io.Reader) ([]byte, error) {
 // ReadJSON reads a request body according to an interface and limits the size to 10MB
 func ReadJSON(ctx context.Context, body io.Reader, intr interface{}) error {
 	logger := logging.Logger(ctx, "requestutils.ReadJSON")
+	if body == nil {
+		return errorutils.New(errors.New("body is nil"), "Error in request body", nil)
+	}
```

Comment:
I added this because when I made POST requests without any body it resulted in a server crash.